### ISSUE TITLE
Regression: kn ssh not working anymore

### ISSUE THREAD

**Issue by sneumann** [phnmnl/KubeNow-plugin#21]
Hi, `kn ssh` is not working anymore, while a manual `ssh -i ssh_key ubuntu@129.70.51.51` works. 
The inventory file now has 
```
[master]
vrebielefeld-master-000 ansible_host=129.70.51.51 ansible_user=ubuntu private_ip=10.0.0.4
[edge]
vrebielefeld-master-000 ansible_host=129.70.51.51 ansible_user=ubuntu private_ip=10.0.0.4
 ansible_host= ansible_user=ubuntu private_ip=
```
which looks weird. Yours, Steffen

**Comment by andersla** [comment]
Thanks for bug report, I have found the problem and will close this issue when patch is merged.

**Comment by andersla** [comment]
Fixed by kubenow/kubenow#a8dc70ddb3c16cf767f1d5b200627f9aa73c59ca
